Get-NetView.psm1

```diff
@@ -2517,6 +2518,17 @@ function Environment {
                         "Get-CimInstance ""Win32_Processor"" | Format-List -Property *",
                         "systeminfo"
     ExecCommandsAsync -OutDir $dir -File $file -Commands $cmds
+
+    if (-not $SkipWindowsRegistry) {
+        $file = "RegistryFileExportOperationStatus.txt"
+        mkdir "$OutDir\RegistryFiles"
```

Comment:
mkdir is not a standard PS command.  Please use "New-Item".  See examples in other functions which take $dir to create a path and directory.

Reply:
Sure, I will use New-Item instead